### DISCUSSION TITLE
Any list of all 'tw-' variables?

### DISCUSSION THREAD

**Discussion by prpanto** [tailwindlabs/tailwindcss#5932] [Help]
Because the docs don't provide any info about this and its more advance, I'm searching for a list with all 'tw-' vars. If anybody know before start digging...

**Comment by adamwathan** [comment]
Hey! These are considered private API so we don't document them anywhere. You can find them by just searching the codebase for `--tw-` though.

What do you want them for?

**Reply by NikitaIT** [reply]
for set postcss fallback custom props, for example:
```
:root {
--tw-*: *;
}
```
